api/src/infrastructure/data-source-manager.ts

```diff
@@ -72,13 +100,24 @@ export class DataSourceManager {
     }
   }
 
-  private getMockDataSqlCode(): string {
-    const filePath = `data/mock-data.sql`;
+  private async getInitialSectionsSqlCode(): Promise<string> {
+    const sectionsfilePath = `data/sections/sections.json`;
+    this.logger.log(
+      `Loading initial data from "${sectionsfilePath}"`,
+      this.constructor.name,
+    );
+    const sections =
+      await SectionsJSONParser.parseSectionsFromFile(sectionsfilePath);
+    return this.sqlAdapter.generateSqlFromSections(sections);
```

Comment:
Please as mentioned, deprecate the sql magic string thingy. Since sections are coming from the sections.json file, with attached baseWidgets, we can just pass it to a repository, and .save() it, removing a lot of complexity